### ISSUE TITLE
As a user, I want to be able to ingest a directory of data that is not part of a bundle

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/harvest#45]
...so I can test and play around with some data in the registry.

**Acceptance Criteria**
TBD

**Notes**
TBD

**Comment by jordanpadams** [comment]
@tdddblog how long do you think this will take (estimate)? I picked 3 work-days, but feel free to revise (may need to add new estimate numbers).

**Comment by tloubrieu-jpl** [comment]
Hi @tdddblog , could you tell me how the configuration file gets the directory instead of the bundle ? I was not able to find that on my own. Thanks

**Comment by tdddblog** [comment]
Use the following section in harvest configuration file. You can provide one or more paths. 
```
  <directories>
    <path>/tmp/dir1/</path>
  </directories>
```
You could not have bundle and directories section at the same time.

**Comment by tloubrieu-jpl** [comment]
Thanks @tdddblog

**Comment by jordanpadams** [comment]
closed per #46
